### ISSUE TITLE
Incorrect text in H1 and title tag of test plan review pages

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#789]
The H1 and title tag of test plan review pages is using the name of the test plan directory instead of the title of the test plan. This creates inconsistency between the names of the links that opens the pages and the title of the target of the link. This inconsistency is a failure in accessibility requirements. It also creates confusion because it is inconsistent with the title of the test plan version history page.

The format of the H1 and title tag element should be :

>Review Test Plan for TEST_PLAN_TITLE

Examples:

The title of the review page for [Checkbox Example (Mixed-State)](https://aria-at-staging.w3.org/test-review/b3d0576a2901ea7f100f49a994b64edbecf81cff/checkbox-tri-state) is:

>Test plan review for pattern: checkbox-tri-state

But it should be:

>Review Test Plan for Checkbox Example (Mixed-State)

The title of the review page for [Color Viewer Slider](https://aria-at-staging.w3.org/test-review/1c6ef2fbef5fc056c622c802bebedaa14f2c8d40/horizontal-slider) is:

>Test plan review for pattern: horizontal-slider

But it should be:

>Review Test Plan for Color Viewer Slider
